Homework/LoggerAsync/App.cs

```diff
@@ -0,0 +1,53 @@
+using LoggerAsync.Enums;
+using LoggerAsync.EventHandlers;
+using LoggerAsync.Services.Abstractions;
+
+namespace LoggerAsync;
+
+public class App
+{
+    private const int IterationCount = 250;
+    
+    private readonly ILoggerService _loggerService;
+
+    public App(
+        ILoggerService loggerService,
+        LogBackupEventHandler backupHandler)
+    {
+        _loggerService = loggerService;
+        _loggerService.BackupSignal += backupHandler.Handle;
+    }
+    
+    public async Task Start()
+    {
+        try
+        {
+            await Task.WhenAll(FirstMethod(), SecondMethod());
+        }
+        catch (Exception e)
+        {
+            Console.WriteLine(e);
+        }
+        
+        Console.ReadLine();
+    }
+
+    private async Task FirstMethod()
+    {
+        await ProcessLog(LogType.Warning);
+    }
+
+    private async Task SecondMethod()
+    {
+        await ProcessLog(LogType.Info);
+    }
```

Comment:
For what you need to create two separate method for this? What if you need 10 tasks to run at the same time? You can pass  ProcessLog method inside WhenAll right away